core/tests/test_utils.py

```diff
@@ -59,6 +62,31 @@ def empty_environ():
     os.environ['DEFAULT_VERSION_HOSTNAME'] = '%s:%s' % (
         os.environ['HTTP_HOST'], os.environ['SERVER_PORT'])
 
+class URLFetchServiceMock(apiproxy_stub.APIProxyStub):
+    """Mock for google.appengine.api.urlfetch"""
+
+    def __init__(self, service_name='urlfetch'):
+        super(URLFetchServiceMock, self).__init__(service_name)
+        self.return_values = {}
+        self.response = None
+        self.request = None
+
+    def set_return_values(self, **kwargs):
```

Comment:
Is it possible to make **kwargs explicit here (i.e. just content, status_code, headers)?